src/TheNextWeek/ray.h

```diff
@@ -18,12 +18,11 @@ class ray {
   public:
     ray() {}
 
-    ray(const point3& origin, const vec3& direction) : orig(origin), dir(direction), tm(0)
-    {}
+    ray(const point3& origin, const vec3& direction)
+      : orig(origin), dir(direction), tm(0) {}
```

Comment:
nit: I suggest updating the formatting in https://github.com/RayTracing/raytracing.github.io/blob/dev/books/RayTracingTheNextWeek.html#L74 to match the listing.

Reply:
Are you talking about `ray.h`, or all other classes?

Reply:
Oops, I managed to misread the file name. Carry on!